api/client/workflows/types.go

```diff
@@ -0,0 +1,103 @@
+/*
```

Comment:
optional: I'd remove this whole abstraction layer
it's not clear to me why we need it, since the rest of this Go module is tightly coupled to the gRPC API definitions